app/assets/javascripts/host_edit_interfaces.js

```diff
@@ -29,6 +29,7 @@ function show_interface_modal(modal_content) {
   modal_window.modal({'show': true});
 
   modal_window.find('a[rel="popover-modal"]').popover({html: true});
+  $('fieldset.compute_attributes select').select2();
```

Comment:
I think it make sense to not do the multi selects, and mostly in places
where the options might be long? - we could add it to other selects like
environment, proxies, domains, subnets etc?